CONTRIBUTING.md

```diff
@@ -17,48 +17,62 @@ articulated in the [Python Community Code of Conduct](https://www.python.org/psf
 
 ### Setup
 
-Run the following:
+#### (1) Clone the mypy repository and enter into it
 ```
-# Clone the mypy repository
 git clone https://github.com/python/mypy.git
-
-# Enter the repository
 cd mypy
+```
 
-# Create then activate a virtual environment
+#### (2) Create then activate a virtual environment
+```
+# On Windows, the commands may be slightly different. For more details, see
+# https://docs.python.org/3/library/venv.html#creating-virtual-environments
 python3 -m venv venv
 source venv/bin/activate
+```
 
-# Install the test requirements and the project
+#### (3) Install the test requirements and the project
+```
 python3 -m pip install -r test-requirements.txt
 python3 -m pip install -e .
-hash -r
+hash -r  # This command isn't necessary if you're on Windows
 ```
 
 ### Running tests
 
-Once setup, you should be able to run tests:
+Running the full test suite can take a while, and usually isn't necessary when
+preparing a PR. Once you file a PR, the full test suite will run on GitHub.
+You'll then be able to see any test failures, and make any necessary changes to
+your PR.
+
+However, if you wish to do so, you can run the full test suite
+like this:
 ```
 python3 runtests.py
 ```
 
-To use mypy to check mypy's own code, run:
+Some magic incantations for running tests include:
 ```
+# Use mypy to check mypy's own code
 python3 runtests.py self
```

Comment:
I'd like to keep the "equivalent" command. People often try to run mypy on mypy directly and it's useful to know what the command to do so is.

Reply:
I axed it for brevity, but fair enough, I'll add it back!